packages/app/app/layout.tsx

```diff
@@ -1,19 +1,41 @@
+"use client";
+
 import { PropsWithChildren } from "react";
+import { Navbar } from "@/components";
+import { WagmiConfig, createConfig } from "wagmi";
+import { ConnectKitProvider, getDefaultConfig } from "connectkit";
+import { gnosis, mainnet } from "wagmi/chains";
 import "../styles/global.css";
-import Navbar from "@/components/navbar/Navbar";
 
-export const metadata = {
-  title: "Stackly | Stack crypto over time.",
-  description:
-    "Stackly is a simple, non-custodial tool that uses the CoW protocol to place recurring swaps based on DCA..",
-};
+const chains = [gnosis, mainnet];
+
+const config = createConfig(
+  getDefaultConfig({
+    // Required API Keys
+    alchemyId: process.env.ALCHEMY_KEY, // or infuraId
+    walletConnectProjectId: "", //process.env.WALLETCONNECT_PROJECT_ID,
+
+    // Required
+    appName: "Stackly",
+
+    // Optional
+    appDescription: "Stack crypto over time.",
+    appUrl: "https://stackly.eth.limo",
+    //appLogo: "https://stackly.eth.limo/favicon.ico", // your app's logo,no bigger than 1024x1024px (max. 1MB)
+    chains,
+  })
+);
```

Comment:
Don't feel this code belongs here.

Maybe we can have a providers file? or something like that and extract this code there.

Reply:
Moved to providers folder